LayoutTests/svg/dom/svg2-inheritance-expected.txt

```diff
@@ -5,7 +5,7 @@ On success, you will see a series of "PASS" messages, followed by "TEST COMPLETE
 
 PASS SVGAElement inherits SVGGraphicsElement
 PASS SVGAngle inherits Object
-PASS SVGAnimateColorElement inherits SVGAnimationElement
+FAIL SVGAnimateColorElement is not defined
```

Comment:
Can you change the test so this doesn't appear as a failure? (either remove the subtest or change to test that it's not defined)

Reply:
I have separate bug already to fix this because we have multiple such cases in this tests and @rwlbuis raised also good question whether we have WPT test for this or we can make one. So I thought to tackle it separately.

Reply:
I don't think we should tackle that completely separately. We want to be able to revert without much hassle.

Reply:
Removed.